examples/recipes/multiple-data-sources/queries/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# JWT decode function https://gist.github.com/a-magdy/a771f1426043ab4b66b19cc9a652908b
```

Comment:
Do we really need this function when, on Unix machines, we have the `base64` utility? See https://askubuntu.com/a/178546

Reply:
`base64` returns error `base64: truncated base64 input` for JWT. 
We can use `echo "$avocadoToken" | base64 -d | jq .` and show something like this, but it looks bad 
<img width="418" alt="image" src="https://user-images.githubusercontent.com/18264667/131451976-7571ebd1-00d8-4632-877a-bde11227b845.png">

<img width="964" alt="image" src="https://user-images.githubusercontent.com/18264667/131452409-d389ca83-34fc-419b-9030-655204d39c7a.png">